homeassistant/components/overkiz/binary_sensor.py

```diff
@@ -0,0 +1,151 @@
+"""Support for Overkiz binary sensors."""
+from __future__ import annotations
+
+from collections.abc import Callable
+from dataclasses import dataclass
+
+from pyoverkiz.enums import OverkizCommandParam, OverkizState
+
+from homeassistant.components.binary_sensor import (
+    BinarySensorDeviceClass,
+    BinarySensorEntity,
+    BinarySensorEntityDescription,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from . import HomeAssistantOverkizData
+from .const import DOMAIN, IGNORED_OVERKIZ_DEVICES
+from .entity import OverkizDescriptiveEntity
+
+
+@dataclass
+class OverkizBinarySensorDescriptionMixin:
+    """Define an entity description mixin for binary sensor entities."""
+
+    is_on: Callable[[str], bool]
```

Comment:
We usually name this `value_fn`

Reply:
Will rename. What does value_fn stand for?

Reply:
c4c6e90

Reply:
Value function